package.json

```diff
@@ -24,5 +24,8 @@
   "homepage": "https://github.com/cs130-w21/19#readme",
   "devDependencies": {
     "concurrently": "^5.3.0"
+  },
+  "dependencies": {
+    "node-cron": "^2.0.3"
```

Comment:
This shouldn't be in the root level `package.json`. This should be in `server/package.json`.